addons/block-cherry-picking/addon.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Grab single block with Ctrl key",
-  "description": "Adds the ability to drag a single block out of the middle of a script (instead of the entire stack attached below it) while holding the Ctrl key.",
+  "name": "Single block grabbing",
+  "description": "Allows you to hold Ctrl to drag a single block out of the middle of a script (instead of the entire stack attached below it).",
```

Comment:
Does it make sense to change this one but not "Duplicate script with Alt key"?

Reply:
I should probably change that one too.